src/site/markdown/server/manage-webadmin.md

```diff
@@ -58,6 +63,18 @@ Response codes:
  - 200: All checks have answered with a Healthy status
  - 500: At least one check have answered with a Unhealthy or Degraded status
 
+ ### List all health checks
+
+ This endpoint lists all the available health checks.
+
+```
+curl -XGET http://ip:port/healthcheck/checks
+```
```

Comment:
I would find it helpful to have a sample response body as part of the doc so that as a user I know what to expect when calling this endpoint